app/controllers/navigator/my_modules_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Navigator
+  class MyModulesController < ApplicationController
+    before_action :load_my_module
+    before_action :check_read_permissions
+
+    include NavigatorActions
+
+    def tree
+      my_modules = my_module_level(@experiment, params[:archived] == 'true')
+      experiments = experiment_level(@experiment.project, params[:archived] == 'true')
```

Comment:
these should all be called something like `experiment_tree`, as they are not just a collection of experiment models, for example